modules/applicationinsights.bicep

```diff
@@ -1,14 +1,15 @@
 // This template is used to create Application Insights.
```

Comment:
I liked the comments you put in the other modules - it looks much cleaner - you could add that here as well.

`// Creates Application Insights Workspace` 
(I don't know the right word for App Insights - workspace, area, resource??)

Reply:
Yeah.. instance possibly?